### PR TITLE
Update example

### PR DESCRIPTION
**Pull request by supercgor**
For some example, reduce the training function and testing function because it used up lots of memories.

### CODE REVIEW

**Comment by lululxvi** [comment]
> For some example, reduce the training function and testing function because it used up lots of memories.

How much memory? All the examples can run well with tensorflow backend without memory issues.

**Comment by supercgor** [comment]
if the number of functions is about 1000, 4GB memory is not enough for pytorch.
the problem only happens in non-cartesian products.

**Comment by lululxvi** [comment]
4GB memory is small. Use the original number.

File: examples/operator/advection_aligned_pideeponet.py [hunk 1]
```diff
@@ -30,31 +30,31 @@ def func_ic(x, v):
...
 )
 
 
 def periodic(x):
     x, t = x[:, :1], x[:, 1:]
-    x *= 2 * np.pi
-    return tf.concat(
-        [tf.math.cos(x), tf.math.sin(x), tf.math.cos(2 * x), tf.math.sin(2 * x), t], 1
+    x = 2 * np.pi * x
+    return bkd.concat(
```
[the remark this answers is not in the document]
Comment (supercgor) [reply examples/operator/advection_aligned_pideeponet.py:48]: May I ask why you don't recommend using `bkd`? I saw other examples using `if` to switch the cases, but the codes are so redundant.

**Pull request closed without merging**